### PR TITLE
Port block mappings

### PR DESCRIPTION
**Pull request by IotaBread**

### CODE REVIEW

File: mappings/net/minecraft/block/Block.mapping [hunk 1]
```diff
@@ -296,3 +301,6 @@ CLASS net/minecraft/unmapped/C_mmxmpdoq net/minecraft/block/Block
 			ARG 3 facing
 		METHOD equals (Ljava/lang/Object;)Z
 			ARG 1 o
+	CLASS C_znvnetlf
+		METHOD rehash (I)V
+			ARG 1 newN
```
Comment (NoComment1105) [inline mappings/net/minecraft/block/Block.mapping]: Is this supposed to have an `N` after it?

**Review by NoComment1105 — CHANGES_REQUESTED** [review]
just the one comment, otherwise good

File: mappings/net/minecraft/block/Block.mapping (hunk 1 above)
Comment (IotaBread) [reply mappings/net/minecraft/block/Block.mapping]: I guess so, as in 'new number'. I should change it though

Comment (NoComment1105) [reply mappings/net/minecraft/block/Block.mapping]: Ah I see, something like `newNumber` or `newNo` would work

Comment (IotaBread) [reply mappings/net/minecraft/block/Block.mapping]: The method is actually from `Object2ByteLinkedOpenHashMap`, and the parameter name comes from there

Comment (NoComment1105) [reply mappings/net/minecraft/block/Block.mapping]: Ah I see. Not something you're thinking of changing then?

Comment (IotaBread) [reply mappings/net/minecraft/block/Block.mapping]: I changed it to `newSize`

**Review by NoComment1105 — APPROVED** [review]
All good from me

**Review by Jamalam360 — APPROVED** [review]

**Review by Jamalam360 — APPROVED** [review]
The changes since my last review all look fine :+1:

**Review by OroArmor — APPROVED** [review]

**Pull request merged**